spec/export_spec.rb

```diff
@@ -64,6 +69,15 @@
       File.delete "#{dir}/issue-1.json"
       File.delete "#{dir}/issue-2.json"
     end
+
+    it 'exports closed issues when that flag is set' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.